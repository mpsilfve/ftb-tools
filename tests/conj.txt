sillä sekä joko niin ettäs paitsi myös
