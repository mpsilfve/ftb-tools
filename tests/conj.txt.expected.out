
"<sillä>"
	"se" Pron Dem Sg Ade abc #1
	"sillä" CC abc #1
"<sekä>"
	"sekä" CC abc #1
	"sekä" CCM abc #1
"<joko>"
	"jo" Adv Ko abc #1
	"joko" CC abc #1
	"joko" CCM abc #1
"<niin>"
	"ne" Pron Dem Pl Ins abc #1
	"niin" Adv Dem abc #1
	"niin" Adv abc #1
	"niin" CCM abc #1
"<ettäs>"
	"ettäs" Adv abc #1
"<paitsi>"
	"paitsi" Adp abc #1
	"paitsi" Adv abc #1
	"paitsi" CCM abc #1
"<myös>"
	"myödä" V Act Imprt Sg2 S abc #1
	"myös" Adv abc #1
	"myös" CC abc #1
